Formalisation(shacl)/Core/PiecesShape/DatasetSeries.ttl

```diff
@@ -74,8 +74,8 @@
   ],
   [
     sh:path dct:publisher ;
-    #sh:class foaf:Agent ;
-    sh:node :AgentShape ;
+    sh:class foaf:Agent ;
+    #sh:node :AgentShape ;
```

Comment:
What is the intent here? This requires an IRI to be entered by the user, but will not allow them to fill in the nested fields. What do we want the user to enter?